pulsar-client-cpp/wireshark/README.md

```diff
@@ -0,0 +1,84 @@
+# Pulsar Wireshark dissector
+
+The Pulsar Wireshark dissector allows to automatically decode the Pulsar binary protocol
+and visualize useful debug information (linking requests with responses, latency stats, etc.)
+
+### Install Wireshark
+
+#### macOS
+
+Recommend use this way to install Wireshark in macOS.
+
+```bash
+brew install homebrew/cask/wireshark
+```
+
+#### Ubuntu
+
+```bash
+sudo apt install wireshark
+```
+
+### Install dependencies
+
+Install Wireshark with the development headers, necessary in order to build the plugin.
+
+**NOTE**: Please make sure the Wireshark application version is the same as the Wireshark Headers version.
```

Comment:
```suggestion
> **NOTE**
> Make sure the Wireshark application version is the same as the Wireshark headers version.
```

Do not use **please** in technical information. Technical information requires an authoritative tone. Terms of politeness are superfluous, convey the wrong tone for technical material, and are not regarded the same way in all cultures. In marketing information, terms of politeness might be appropriate. Use the imperative mood in the first sentence of each step.

Details see https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.x24xr2yx9b9t

Reply:
I see. I'll read this document carefully. Thanks for your suggestion.